Package.swift

```diff
@@ -22,6 +22,11 @@ let package = Package(
             dependencies: []),
```

Comment:
Ideally we'd force the vended target to conform to include `strict-concurrency=complete`, but [doing so would mark this repo as having unsafe build flags](https://stackoverflow.com/questions/74986278/cannot-use-package-as-dependency-with-concurrency-checking-enabled-in-package-sw). Once we can use `enableUpcomingFeature` (see below), this will get better.

Reply:
OK I see that we use unsafe build flags in the test target and that's OK since it's not being consumed by other packages.